src/FakeItEasy/Properties/AssemblyInfo.cs

```diff
@@ -5,6 +5,7 @@
 [assembly: CLSCompliant(true)]
 
 [assembly: InternalsVisibleTo("DynamicProxyGenAssembly2, PublicKey=0024000004800000940000000602000000240000525341310004000001000100c547cac37abd99c8db225ef2f6c8a3602f3b3606cc9891605d02baa56104f4cfc0734aa39b93bf7852f7d9266654753cc297e7d2edfe0bac1cdcf9f717241550e0a7b191195b7667bb4f64bcb8e2121380fd1d9d46ad2d92d2d15605093924cceaf74c4861eff62abf69b9291ed0a340e113be11e6a7d3113e92484cf7045cc7")]
+[assembly: InternalsVisibleTo("FakeItEasy.Extensions.ValueTask, PublicKey=00240000048000009400000006020000002400005253413100040000010001002f50d82092713be482c885861c984334606da0f54c78738a3dd0862fb2dfc6080f0780132cc65d88f0f0c70af74e8a53430962395bfc1a36fab08b7a2549d387e805c13cc84acd884447ec8c4dcfb6216df720f0998380f9c906b5de8141798d64661f036d47274e6ecb76c9cde5f4cf2b521040601e44b3914fbeb9f39127f9")]

```

Comment:
On one hand, I'm not very happy about that, but on the other hand, I can't think of any real drawback. FakeItEasy.Extensions.ValueTask is a first-party package, so maybe its fine if it has a privileged status compared to third-party ones? Anyway, it makes things a bit simpler and avoids some code duplication.